html/.storybook/preview-head.html

```diff
@@ -0,0 +1,25 @@
+<script>
+
+  const runOnPageChange = () => {
+    getNogoLinks = Array.from(document.querySelectorAll('a[href="#nogo"]'));
+    if (getNogoLinks.length > 0) {
```

Comment:
i dont think you need the length check here, since forEach would just run 0 times on length = 0.